pipelines/azure_pipelines_nightly.yml

```
@@ -1,109 +1,288 @@
-              --filelist "$(Pipeline.Workspace)\configs\filelist.txt" `
-              --user "$(SignClientUser)" `
-              --secret "$(SignClientSecret)" `
-              --name "Code Sign packages" `
-              --description "Signing packages" `
-              --descriptionUrl "https://github.com/OData/WebApi"
-            displayName: Sign packages
-              
-          - publish: $(Pipeline.Workspace)/BuildPackages
-            displayName: Publish Signed Packages
-            artifact: SignedPackages
+      name: MSSecurity-1ES-Build-Agents-Pool
+      image: MSSecurity-1ES-Windows-2019
+      os: windows
+    customBuildTags:
+    - ES365AIMigrationTooling
+    stages:
+    - stage: build
+      jobs:
+      - job: Main
+        displayName: Main Build
+        timeoutInMinutes: 300
+        templateContext:
+          outputs:
+          - output: pipelineArtifact
+            displayName: 'Publish SBOM'
+            targetPath: $(Build.ArtifactStagingDirectory)\sbom
+          - output: pipelineArtifact
+            displayName: 'Publish Artifact - Nuget Packages'
+            targetPath: $(Build.ArtifactStagingDirectory)\Nuget
+            artifactName: NuGetDrop
+          - output: nuget
+            displayName: 'NuGet push - Nightly packages to MyGet'
+            packageParentPath: '$(Build.ArtifactStagingDirectory)'
+            packagesToPush: $(Build.ArtifactStagingDirectory)\Nuget\*Nightly*.nupkg
+            nuGetFeedType: external
+            publishFeedCredentials: 'MyGet.org - AspNetCoreOData Feed'
+        steps:
+        - checkout: self
+        - template: pipelines/task-group-policheck-v1.yml@self
+        - task: NuGetToolInstaller@1
+          displayName: Use NuGet >=5.8.0
+          inputs:
+            versionSpec: '>=5.8.0'
+            checkLatest: true
+        - task: UseDotNet@2
+          displayName: Use .NET Core sdk 2.x
+          inputs:
+            version: 2.x
+        - task: UseDotNet@2
+          displayName: Use .NET Core sdk 3.x
+          inputs:
+            version: 3.x
+        - task: NuGetCommand@2
+          displayName: NuGet restore *\*.sln
+          inputs:
+            solution: sln/WebApiOData.AspNet.sln;sln/WebApiOData.AspNetCore.sln;sln/WebApiOData.E2E.AspNet.sln;sln/WebApiOData.E2E.AspNetCore.sln
+        - task: VSBuild@1
+          displayName: Build solution sln\WebApiOData.AspNet.sln
+          inputs:
+            solution: 'sln\WebApiOData.AspNet.sln '
+            platform: $(BuildPlatform)
+            configuration: $(BuildConfiguration)
+        - task: VSBuild@1
+          displayName: Build solution sln\WebApiOData.E2E.AspNet.sln
+          inputs:
+            solution: sln\WebApiOData.E2E.AspNet.sln
+            platform: $(BuildPlatform)
+            configuration: $(BuildConfiguration)
+        - task: DotNetCoreCLI@2
+          displayName: 'build Microsoft.AspNetCore.OData.csproj '
```

nit:
```suggestion
          displayName: 'Build Microsoft.AspNetCore.OData.csproj '
```
Most other `displayName` start with upper case